src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -321,8 +321,12 @@ private async Task<long> CreateGlobalStateTableAsync(SqlConnection connection, S
                         UserFunctionID char(16) NOT NULL,
                         UserTableID int NOT NULL,
                         LastSyncVersion bigint NOT NULL,
+                        LastAccessTime Datetime NULL,
                         PRIMARY KEY (UserFunctionID, UserTableID)
                     );
+                ELSE IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE Name = N'LastAccessTime'
```

Comment:
I would like for us to have an explicit test for a lease table where the LastAccessTime column doesn't exist to validate that the `ELSE IF NOT EXISTS` code path works as intended.

(Is this possible, by the way, in the real world?)

Reply:
It will be for people with existing functions that just update the extension version but everything else stays the same. But that'll drop off as we march towards GA since after this change gets in no one should have that going forward.

Reply:
I agree with @Charles here, but I can look into adding a test to be sure nothing breaks in the meantime.